Fix failure to find libc++sbi.so when compiling with LLVM/Clang

Downstream packages like iceoryx-posh were failing to build because cpptoml
was exporting absolute paths from the host in the CMake target
configuration files. This was caused by a condition that used MeTA-cmake
to find libcxx when Clang is used.  Removing this conditional block allows
cpptoml to build for both GCC and Clang.

Upstream-Status: Inactive-Upstream

Signed-off-by: Alexander Entinger <consulting@lxrobotics.com>

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4ec25cc..802d1a9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,17 +20,6 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/deps/meta-cmake)

 if(UNIX OR MINGW)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -pedantic")
-
-  if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-    if(CMAKE_GENERATOR STREQUAL "Ninja")
-      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fcolor-diagnostics")
-    endif()
-
-    if(ENABLE_LIBCXX)
-      find_package(LIBCXX REQUIRED)
-      set_libcxx_required_flags()
-    endif()
-  endif()
 endif()

 set(CMAKE_REQUIRED_FLAGS "${CMAKE_REQUIRED_FLAGS} ${CMAKE_CXX11_STANDARD_COMPILE_OPTION}")
